cores/esp8266/core_esp8266_postmortem.cpp

```diff
@@ -249,15 +256,23 @@ static void postmortem_report(uint32_t sp_dump) {
         sp_dump = stack_thunk_get_cont_sp();
     }
 
-    if (sp_dump > cont_stack_start && sp_dump < cont_stack_end) {
+    uint32_t stack_end;
+
+    // above and inside of cont, dump from the sp to the bottom of the stack
+    if (rst_info.reason == REASON_USER_STACK_OVERFLOW) {
+        ets_printf_P(PSTR("\nctx: cont\n"));
+        stack_end = cont_stack_end;
```

Comment:
two conditions for the same code, is it intended ?

Reply:
It is not, leftover from old impl... Changed to 'cond1 || cond2'